src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -526,6 +543,8 @@ export class PatientViewPageStore {
                         validateMutationalSignatureRawData(genericAssayRawData)
                     ) {
                         return Promise.resolve(genericAssayRawData);
+                    } else {
+                        return Promise.resolve(genericAssayRawData);
```

Comment:
This `if` seems useless. If not valid, should we return an empty result or something?

Reply:
I think the function `validateMutationalSignatureRawData` and the if/else statement are not necessary anymore. We have added the option to upload mutational signature data without p-values. This is needed for some tools that do provide contribution and count data but no p-values.

So now we can return the `genericAssayRawData` if `mutationalSignatureMolecularProfileIds.length > 0` or `[]`